src/Powercord/plugins/pc-moduleManager/util/misc.js

```diff
@@ -1,3 +1,5 @@
+exports.REPO_URL_REGEX = /https?:\/\/(?:www\.)?github\.com\/([^/\s]+)\/([^/\s]+)(?:\/tree\/(\S+))?/;
```

Comment:
```suggestion
exports.REPO_URL_REGEX = /https?:\/\/(?:www\.)?github\.com\/(\S+)\/(\S+)(?:\/tree\/(\S+))?/;
```

Reply:
That also includes slashes though

Reply:
oh I didn't see that. You don't need to include slashes though, you can use `\S+?`. ? makes it non greedy

Reply:
![Screenshot on 2022-07-26 at 12 44 04](https://user-images.githubusercontent.com/14863373/181075196-890e9842-4d8c-4da0-95ed-3ac7260521a0.png)
Making it non-greedy causes it to not go all the way to the end.